src/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --button-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 15px 5px 45px;
+}
+
+.card__name {
+  margin-bottom: 6px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  margin-bottom: 15px;
+}
+
+.rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.rate__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 15px;
+  margin-left: -4px;
```

Comment:
better not to use '-' margins and remove 'margin-left' from your `.rate_stars:first-child`